internal/callbacker/dispatcher.go

```diff
@@ -38,12 +44,28 @@ func (d *CallbackDispatcher) GracefulStop() {
 	}
 }
 
+func (d *CallbackDispatcher) Init() error {
+	const bundleSize = 100
```

Comment:
I think for consistency to other services we could call this `batchSize`

Reply:
changed